Disfunction/Assets/DamageSystem.cs

```diff
@@ -0,0 +1,27 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class DamageSystem : MonoBehaviour
+{
+
+    public GameObject obj;
+    public float health;
+
+    public GameObject dummy;
+
+    private void Start()
+    {
+        spawn(health);
+    }
+
+    public void spawn(float health)
+    {
+        float x = Random.Range(-3,5);
```

Comment:
Make the X and Y range values public global variables, so we can edit in inspector.